coriolisclient/tests/cli/test_licensing.py

```diff
@@ -0,0 +1,386 @@
+# Copyright 2024 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from cliff import command
+from cliff import lister
+from cliff import show
+from unittest import mock
```

Comment:
`cliff` is a third-party package, while `unittest` is standard. Please separate their imports, and apply everywhere:
```suggestion

from unittest import mock

from cliff import command
from cliff import lister
from cliff import show
```

Reply:
Please apply this to all the files. Other than that, this LGTM